src/main/java/com/openlattice/datastore/services/EdmService.java

```diff
@@ -1628,6 +1629,28 @@ public void updateEntitySetPropertyMetadata( UUID entitySetId, UUID propertyType
         entitySetPropertyMetadata.executeOnKey( key, new UpdateEntitySetPropertyMetadataProcessor( update ) );
     }
 
+    @Override
+    public Map<UUID, EntityTypePropertyMetadata> getAllEntityTypePropertyMetadata( UUID entityTypeId ) {
+        return getEntityType( entityTypeId ).getProperties().stream()
```

Comment:
Same comment here on the hazelcast load, since `getEntityTypePropertyMetadata` reads from hazelcast -- it would be faster to use a Predicate on `entityTypeId` (and add a map config in the mapstore)